roles/graylog_server/tasks/main.yml

```
@@ -1,29 +1,28 @@
 - block:
 
-  - name: "install graylog server"
+  - name: "install graylog-server"
```

just for consistency :)
```suggestion
  - name: 'install graylog-server'
```